nav-app/src/app/exporter/exporter.component.ts

```diff
@@ -478,6 +480,10 @@ export class ExporterComponent implements AfterContentInit {
             if (self.showName() || self.showDescription()) {
                 let about = {"title": "about", "contents": []};
                 if (self.showName()) about.contents.push({"label": "name", "data": this.viewModel.name});
+                if (self.showDomain()) {
+                    let domain = this.dataService.getDomain(this.viewModel.domainID);
+                    about.contents.push({"label": "domain", "data": domain.name + " " + domain.version})
+                }
```

Comment:
The "about" panel is pretty full with name, domain/version, and description information. We might want to consider moving it over to the "filters" box which currently just has the platforms. We could rename filters to "showing" or something to that effect (definitely should try to come up with a better label than "showing" though).

![image](https://user-images.githubusercontent.com/28868888/100238988-d977a580-2efe-11eb-8654-b8f7f3ac1a8a.png)

Reply:
I like the idea of moving it from the "about" section, so it'll be less cluttered. Since the "filters" section can also contain lot of content, what do you think of adding a new "domain" section to the header?
![image](https://user-images.githubusercontent.com/48832936/100252134-8ce79680-2f0d-11eb-9cc5-466180204f11.png)

Reply:
Fine by me! Probably should also add an option to hide it from the header though, some users might want to reclaim that horizontal space in small image sizes. 

@ArtificialErmine we'll need to make this change in the Python implementation.